runtime/src/accounts_db.rs

```diff
@@ -233,6 +233,9 @@ pub struct AccountStorageEntry {
     /// status corresponding to the storage, lets us know that
     ///  the append_vec, once maxed out, then emptied, can be reclaimed
     count_and_status: RwLock<(usize, AccountStorageStatus)>,
+
+    #[serde(skip)]
+    store_count: AtomicUsize,
```

Comment:
I wanted this part of `count_and_status` but the change require alot change. And I don't need absolute accuracy for `store_count`. 
So, this might be inconsistent with `count_and_status` due to lack of atomicity / synchronization with `count_and_status`

Maybe, to indicate such a nature, I'll rename this to `approx_store_count`?

Reply:
Yea, a comment would be nice to indicate how it's different to `count_and_status`